package.json

```diff
@@ -20,7 +20,7 @@
     "pretest": "node tools/test-local.js",
     "test": "yarn test:staging",
     "posttest": "node tools/revert-test-local.js",
-    "test:staging": "yarn run cypress run",
+    "test:staging": "yarn run cypress run --reporter junit",
```

Comment:
small nit

do we need to do `yarn run cypresss ...`

You could do

```
   "test:watch": "cypress open",
    "test:chrome": "cypress run --browser chrome  --reporter junit",
    "test:firefox": "cypress run --browser firefox  --reporter junit",
    "test": "yarn test:chrome && test:firefox"
```

No need to do that in this PR - and yes cross browser is supported since version 4

Reply:
True, could remove the `yarn run` bit, but not sure the value of running the tests on different browsers for now.